unlock-app/src/components/interface/checkout/main/Confirm.tsx

```diff
@@ -32,14 +33,15 @@ export function Confirm({
     ToastHelper.error(message)
   }
 
-  const onConfirmed = (lock: string, hash?: string) => {
+  const onConfirmed = (lock: string, hash?: string, network?: number) => {
```

Comment:
The conformation screen needs to wait for the transaction to be mined on the "origin" chain, which can be different from the lock's chain so we are passing optionally the network.

Reply:
maybe should be called `transactionNetwork` to be consistent with what is found in the component  ?